src/charm.py

```diff
@@ -29,11 +35,15 @@ def __init__(self, msg: str, status_type=None):
 
 
 class Operator(CharmBase):
+    _stored = StoredState()
+
     def __init__(self, *args):
         super().__init__(*args)
 
         self.log = logging.getLogger(__name__)
         self.image = OCIImageResource(self, "oci-image")
+        self._stored.set_default(hash_salt=_gen_pass())
```

Comment:
I'm guessing this was pulled from one of our other charms that happens to include a password in the configmap, and thus we'd salt the hash to make it less obvious.  In this case where we're just storing details of a sidebar, do we need a salt at all?  Or at least do we need a randomly generated one?